sample/confidential_client_certificate_sample.py

```diff
@@ -12,6 +12,8 @@
     "private_key_file": "filename.pem"
         // For information about generating thumbprint and private key file, refer:
         // https://github.com/AzureAD/microsoft-authentication-library-for-python/wiki/Client-Credentials#client-credentials-with-certificate
+
+    "endpoint": "https://graph.microsoft.com/v1.0/users"
```

Comment:
A different topic here. Currently this line becomes the new last line in this json setting, it would cause a json loading error when customer (Saeed on stage?) do copy&paste and run it for the first time, because the [previous last setting (now it becomes the second-last setting) contains no trailing comma either](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/112/files#diff-3005f568326ab0f67d4ddfd9c3b0a7feR12).

Suggestion: it is better to add new setting in the middle of a json file, not at the end. :-)

Life Hack: In my own json setting files, I would purposely add an extra last setting as `"placeholder": null` BEFOREHAND, so I won't hit that issue at all. But we probably would not want to add that noise in such public-facing sample. So that is not applicable here.